gulp/tests/prod-build/prod-build.teardown.js

```diff
@@ -18,9 +18,18 @@ import {
 
 // Delete the mock files after testing.
 filesToMock.forEach( ( file ) => {
+	// Delete the mock file
 	if ( fs.existsSync( file.dest ) ) {
 		fs.unlinkSync( file.dest );
 	}
+
+	// Check if a previous version of the file exists
+	const existingFile = file.dest.replace( /(\.[\w\d_-]+)$/i, '-existing$1' );
```

Comment:
This regexp could be simplified like this: `/(\.[\w-]+)$/i`
`\w` matches `[a-zA-Z0-9_]`, we don't need `\d` and `_`.

Reply:
@benoitchantre updated

Reply:
Thank you for the changes.